lib/git_chain/commands/delete.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+require "optparse"
+
+module GitChain
+  module Commands
+    class Delete < Command
+      include Options::ChainName
+
+      def description
+        "Delete chain"
+      end
+
+      def run(options)
+        if Git.rebase_in_progress?
+          raise(Abort, "A rebase is in progress. Please finish the rebase first and run 'git chain rebase' after.")
+        end
+
+        chain = current_chain(options)
+
+        puts_debug("Deleting chain #{chain.formatted}}}")
+
+        branches = chain.branches
+        branches.each do |b|
+          Git.set_config("branch.#{b}.chain", nil, scope: :local)
+          Git.set_config("branch.#{b}.parentBranch", nil, scope: :local)
+          Git.set_config("branch.#{b}.branchPoint", nil, scope: :local)
+        end
+  
+        unless branches.empty?
+          puts_warning("Removed #{branches.map { |b| "{{info:#{b}}}" }.join(", ")} from the chain.")
```

Comment:
This prints:
<img width="1032" alt="image" src="https://user-images.githubusercontent.com/1216046/217902999-241e19d6-6b84-44dc-b36a-0ac0d93fed91.png">

I’m not even sure it's needed, the `puts_success` below seems to cover it already.

It is a warning in `Setup` because it highlights that some branches may mistakenly have been removed.